src/System.Private.ServiceModel/tests/Common/Infrastructure/xunit/IssueAttribute.cs

```diff
@@ -22,6 +23,9 @@ namespace Infrastructure.Common
     [AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
     public class IssueAttribute : WcfSkippableAttribute
     {
+        private static bool s_includeAllIssues = DetectIssuesToInclude();
+        private static HashSet<int> s_includeIssuesHash;
```

Comment:
I think the code is correct. It's just these skip test, skip issue, include test, include issue logics which together made my head hurt 😄.

I am wondering if it will be easier to understand if we keep only one variable:
`private static HashSet<int> s_IgnoredIssues = DetectIgnoredIssues();`

and an issue will be ignored based on logic below:
-  `s_IgnoredIssues==null`: No issues will be ignored. This is default behavior.
-  `s_IgnoredIssues.Count==0`: All issues will be ignored. The is the behavior of IncludeTestsWithIssues=all.
-  `s_IgnoredIssues.Count>0` : Only specific issues will be ignored.

An issue is ignored means the inclusion of a test will not be affected by the issue in any way.